src/Akka.Persistence.Azure.TestHelpers/Akka.Persistence.Azure.TestHelpers.csproj

```diff
@@ -1,11 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\common.props" />
   <PropertyGroup>
-    <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
The reason for the comment here - using the variables from `common.props` makes it simple for us to change versions in our libraries and synchronize them everywhere. We should stick to .NET Standard 2.0 though since that's what Akka.NET supports currently.